olive/passes/pytorch/lora.py

```diff
@@ -620,6 +625,23 @@ def train_and_save_new_model(
             # train
             logger.info("Running fine-tuning")
             train_result = trainer.train()
+
+            # log metrics now only support Azure ML system. Metrics can be found in Metrics tab in Azure ML job
```

Comment:
I don't think manual logging of metrics might be needed. Can you try using `"report_to": "mlflow"` under the training arguments config. 
That should directly log the metrics to mflow in realtime. https://huggingface.co/docs/transformers/v4.38.1/en/main_classes/trainer#transformers.TrainingArguments.report_to

Reply:
Wow good to know this! I tested and it worked. I'll close this PR. thanks!

Reply:
Awesome! thanks for checking